### ISSUE TITLE
Recoil

### ISSUE THREAD

**Issue by Neo42** [simple-icons/simple-icons#5498]
**Brand Name:** Recoil
**Website:** https://recoiljs.org/
**Alexa rank:** 190,141
**GitHub stars:** 12.6k
  

**Official resources for icon and color:** [official svg logo](https://github.com/facebookexperimental/Recoil/blob/c1a899e3e7fa8fc0304f9237f64a208480086bc2/logo/SVG/logo.svg) &  #007af4 
  

I'm on it.

**Comment by Neo42** [comment]
I'm on it.

**Comment by Neo42** [comment]
Sure, please hand it over to someone else willing to give it a go.

**Comment by Sayvai** [comment]
Hey @PeterShaggyNoble / @Neo42,

I've made changes locally to add the Recoil icon to my local `recoil` branch, but i do not have permissions to push up the branch to remote origin, before considering opening a PR to merge in `develop`.

Are either of you able to grant me the relevant privileges?

**Comment by Neo42** [comment]
Hey @LitoMore, would you mind taking a look?

**Comment by LitoMore** [comment]
@Sayvai See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork.

**Comment by Sayvai** [comment]
Thanks @LitoMore.

However, i just figured out there is already an open Pull Request opened for this work at #5998, which surprisingly has been left opened since June 2021 (1.5+. years).

Would you be able to review #5998, and then after an approval, to then merge and release?

cc: @Neo42

**Comment by Neo42** [comment]
@Sayvai Yeah, that pr LGTM but sorry that I don't have the writing access so I can't help you with that. Maybe @LitoMore or @PeterShaggyNoble could lend a hand.

**Comment by young-st511** [comment]
Hi, I'm just waiting for this issue to be solved. Can't I create a new PR for Recoil? I'm a novice, so I don't know much about open-source manners. Sorry🙁

**Comment by ripzerotwo** [comment]
Yeah feel free too its been like a month no update. If you want by the end of today if yhere is no update ill make a pr and fix the issue.

**Comment by ripzerotwo** [comment]
Though tbf it seems that the one open pr is complete just waiting for maintainers to add to update schedule and merge pr.
